src/actinia_core/rest/persistent_processing.py

```diff
@@ -478,22 +479,152 @@ def _lock_target_mapset(self):
+        cur = con.cursor()
+
+        sql_script_folder = os.path.join(os.getenv("GISBASE"), "etc", "sql")
+        drop_view_sql = os.path.join(sql_script_folder, 'drop_views.sql')
+        with open(drop_view_sql, 'r') as sql:
+            sql_drop_str = sql.read()
+        cur.executescript(sql_drop_str)
+
+        view_sql_file_names = [
+            "raster_views.sql",
+            "raster3d_views.sql",
+            "vector_views.sql",
+            "strds_views.sql",
+            "str3ds_views.sql",
+            "stvds_views.sql"
+        ]
+        for view_sql_file_name in view_sql_file_names:
+            view_sql_file = os.path.join(sql_script_folder, view_sql_file_name)
+            with open(view_sql_file, 'r') as sql:
+                sql_view_str = sql.read()
+            cur.executescript(sql_view_str)
+        con.commit()
+        if con:
+            con.close()
+        del cur
+
+    def _merge_tgis_dbs(self, tgis_db_path_1, tgis_db_path_2):
+        """Merge two tgis sqlite.db files
+
+        Args:
+            tgis_db_path_1(str): path of a tgis sqlite.db file in which the
+                                 other should be merged
+            tgis_db_path_2(str): path of a tgis sqlite.db file which should be
+                                 merged in tgis_db_path_1
+        """
+        con = sqlite3.connect(tgis_db_path_1)
+        con.execute(f"ATTACH '{tgis_db_path_2}' as dba")
+        con.execute("BEGIN")
+
+        table_names1 = [row[1] for row in con.execute(
+            "SELECT * FROM sqlite_master where type='table'")]
+        table_names2 = [row[1] for row in con.execute(
+            "SELECT * FROM dba.sqlite_master where type='table'")]
+
+        # merge databases
+        for table in table_names2:
+            if table == 'tgis_metadata':
+                con.execute(f"DROP TABLE {table}")
+                con.execute(f"CREATE TABLE {table} AS "
+                            f"SELECT * FROM dba.{table}")
+                continue
+            # for example raster_register_xxx tables are not in both dbs
+            if table not in table_names1:
+                con.execute(f"CREATE TABLE {table} AS "
+                            f"SELECT * FROM dba.{table}")
+                continue
+            combine = f"INSERT OR IGNORE INTO {table} SELECT * FROM dba.{table}"
+            con.execute(combine)
+        con.commit()
+        con.execute("detach database dba")
+        if con:
+            con.close()
+
+    def _change_mapsetname_in_tgis(self, tgis_path, source_mapset,
+                                   target_mapset, target_tgis_db):
+        """Replaces the mapset name in the tgis sqlite.db
+
+        Args:
+            tgis_path(str): path of the tgis folder in the source mapset
+            source_mapset(str): name of source mapset
+            target_mapset(str): name of target mapset
+
+        Raises:
+            This method will raise an AsyncProcessError if a group has no REF file
```

Comment:
I think this is just copied from above and can be removed

Reply:
Was updated in https://github.com/mundialis/actinia_core/pull/233/commits/7f454e717bcb71019144b5d36e6cfbd1639832e9